<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginForm</class>
 <widget class="QWidget" name="loginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palm Coin</string>
  </property>
  <widget class="QComboBox" name="platformSelector">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="platformLab">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Platform:</string>
   </property>
  </widget>
  <widget class="QLabel" name="accessLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Access Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secretLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Secret Key:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accessInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="secretInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="checkBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
